xcode/Subconscious/Shared/Models/Slug.swift

```diff
@@ -67,6 +67,11 @@ public struct Slug:
         self == Slug.profile
     }
     
+    /// Excludes "internal" slugs like `Slug.profile`
+    public var isListable: Bool {
```

Comment:
Nit: Perhaps `isHidden` (inverse)?

Non-blocking.

Reply:
Changed, added tests to assert the behaviour is working